<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_hapus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>600</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HAPUS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_btsA">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hitung">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>600</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HITUNG HASIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_jumlahpias">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>720</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>JUMLAH PIAS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_hasilintegrasi">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>710</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HASIL INTEGRASI NUMERIK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_btsA">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_btsB">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lebarpias">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hasilpias">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>780</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hasilnumerik">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>780</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>36</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hasilintegrasi_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>720</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GALAT RELATIF</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_galatrelatif">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>780</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_detail">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>870</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LIHAT DETAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_btsA_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_btsA_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>671</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Definisi Fungsi</string>
    </property>
    <widget class="QLabel" name="label_inputfungsi">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>210</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Input Fungsi</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputFungsi">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_inputfungsi_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>210</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Antiturunan</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAntiTurunan">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>631</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameter Integrasi</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>680</y>
      <width>881</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Hasil Perhitungan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>680</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PROGRAM PERHITUNGAN NEWTON COTES: METODE TRAPESIUM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>I Nyoman Yogi Putra Arthawan (2415101060)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>I Gede Nyoman Arya Adi Prima Merta (2415101024)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>pushButton_hapus</zorder>
   <zorder>label_btsA</zorder>
   <zorder>pushButton_hitung</zorder>
   <zorder>label_jumlahpias</zorder>
   <zorder>label_hasilintegrasi</zorder>
   <zorder>lineEdit_btsA</zorder>
   <zorder>lineEdit_btsB</zorder>
   <zorder>lineEdit_lebarpias</zorder>
   <zorder>lineEdit_hasilpias</zorder>
   <zorder>lineEdit_hasilnumerik</zorder>
   <zorder>pushButton_back</zorder>
   <zorder>label_hasilintegrasi_2</zorder>
   <zorder>lineEdit_galatrelatif</zorder>
   <zorder>pushButton_detail</zorder>
   <zorder>label_btsA_2</zorder>
   <zorder>label_btsA_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
